integrations/make.com/integration.definition.ts

```diff
@@ -0,0 +1,35 @@
+import { IntegrationDefinition, messages } from '@botpress/sdk';
+import { z } from 'zod';
+
+const INTEGRATION_NAME = "makecom";
+
+export default new IntegrationDefinition({
+  name: INTEGRATION_NAME,
+  version: '0.2.0', 
+  icon: 'icon.svg', 
+  readme: 'hub.md',
+  configuration: {
+    schema: z.object({
+      webhookUrl: z.string().url().describe('Make.com webhook URL'),
+    }).describe('Configuration schema for Make.com Integration'),
+  },
+  channels: {
```

Comment:
**nitpick:** you can simply set `channels` to be an empty object, i.e. `channels: {},` like in the dall-e integration [here](https://github.com/botpress/botpress/blob/8c6a2e2abc74fc387f6191645587d92b0acc5741/integrations/dalle/integration.definition.ts#L21), channels should only be defined if the integration interacts with another messaging service.